contracts/stackflow.clar

```diff
@@ -29,6 +29,7 @@
 ;; SOFTWARE.
 
 (use-trait sip-010 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
+(impl-trait .stackflow-token.stackflow-token)
```

Comment:
if we want a consistant hash/checksum, should this be the full principal?

Reply:
Ah, you're right. Will fix in 0.3.1. 🙏